server/container/guice/cassandra-guice/src/test/java/org/apache/james/FixingGhostMailboxTest.java

```diff
@@ -194,14 +197,11 @@ void ghostMailboxBugShouldChangeMailboxId() {
     void ghostMailboxBugShouldDiscardOldContent() {
         MailboxId newAliceInbox = mailboxProbe.getMailboxId(MailboxConstants.USER_NAMESPACE, ALICE, MailboxConstants.INBOX);
 
-        given()
-            .header("Authorization", accessToken.serialize())
-            .body("[[\"getMessageList\", {\"filter\":{\"inMailboxes\":[\"" + newAliceInbox.serialize() + "\"]}}, \"#0\"]]")
-        .when()
-            .post("/jmap")
-        .then()
-            .statusCode(200)
-            .body(NAME, equalTo("messageList"))
+        await().timeout(Duration.FIVE_SECONDS)
```

Comment:
Please explain why we should expect a delay here

Reply:
Just a simple thinking. this test fails on CI but it never happens on my machine. I think it might because of the elastic search indexing not done yet. and doing this await, I hope I can resolve the failed test on CI.

Reply:
But we already await after each change we make, we shouldn't have to wait here too

Reply:
Or the previous await is not awaiting the previous partly asynchronous `flush`, despite the loads of query parameters supposed to have that very meaning....

Maybe taking a more generic approach about that issue (rather than modifying individual test cases) could be a good idea.

How about adding a sleep(1s) after the flush within the DockerElasticSearch container?

Reply:
oh shit, I forgot to add the flushing to DockerElasticSearchExtension. Let's see how it's gonna be.